### ISSUE TITLE
[css-highlight-api] How should highlight pseudos be exposed to assistive techonologies?

### ISSUE THREAD

**Issue by sanketj** [w3c/csswg-drafts#4601]
Moving this issue over from: https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/73.

Post from @Reinmar:
I wonder if and how the semantics of created highlights should be exposed to assistive technologies.

For instance – when highlights would be used to render spellchecking squiggles we should also have a way to tell the that this piece of text is misspelled. I don't know how the selection itself is rendered in the Accessibility Object Model, but I think that highlights should be able to use the same mechanism.

I'd imagine that the easiest solution may be to allow defining a highlight's title. For a misspelled fragment of text it could be set to "misspelled" (that's at least what VoiceOver reads).

Post from @Comandeer
In the simplest case something like HighlightRange#label should be enough. However, if you already referenced AOM, there is proposal for virtual accessibility nodes, that are dedicated just for this case: exposing non-element nodes in accessibility tree. In such case HighlightRange interface should be updated in following manner:
```
partial interface HighlightRange {
	AccessibleNode attachAccessibleRoot();
}
```
The only part that it's worrying me a little is the fact that VANs are behaving a little like pseudoelements (they could have dimensions and position).

**Comment by sanketj** [comment]
@frivoal, I don't have permission to add labels (any way for me to get that?). Could you add the highlight API label please? Thanks!

**Comment by frivoal** [comment]
Adding the i18n tracker tag here, since we're talking about making it possible to describe the highlights, likely through some piece of text that would be presented to the user. The mechanism we end up choosing needs to be able to account for internationalized text.

**Comment by dandclark** [comment]
I guess this is a dupe of #6498. @sanketj, I know this one came first, but the more recent discussion has happened over at #6498 so is it OK with you if we close this one in favor of that one?

**Comment by sanketj** [comment]
@frivoal Do you want to keep this issue open for i18n tracking or should just move over to #6498?

**Comment by frivoal** [comment]
Sorry for creating the dup. Yes, given the state of discussions, I think we should close this one and continue over there.
